packages/components/src/Notification/subcomponents/GenericNotification/GenericNotification.module.scss

```diff
@@ -33,14 +33,15 @@
   align-items: center;
   justify-content: center;
 
-  &:focus {
+  &:focus,
+  &:focus-visible {
     outline: none;
   }
 
   &:focus-visible::after {
     content: "";
 
-    $focus-ring-offset: 6px;
```

Comment:
Convert to CSS variable syntax

Reply:
Curious what our strategy is for this?

It seems a bit odd to just change one thing in a file to use CSS syntax while everything else remains SCSS. But I also don't want to blow out the changes in the PR to the point it's hard to tell what's an intentional change and what's just clean up.